src/Pact/Eval.hs

```diff
@@ -112,7 +112,6 @@ enforceKeySetName mi ksn = do
       Just ks -> pure ks
   _ <- computeGas (Left (mi,"enforce keyset name")) (GPostRead (ReadKeySet ksn ks))
   runSysOnly $ enforceKeySet mi (Just ksn) ks
-{-# INLINE enforceKeySetName #-}
```

Comment:
the removal of these inlines seems to be extracurricular. Let's not touch these without a justification

Reply:
Reverted